python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_base_group_chat.py

```diff
@@ -543,9 +575,20 @@ async def save_state(self) -> Mapping[str, Any]:
         self._is_running = True
 
         try:
-            # Save the state of the runtime. This will save the state of the participants and the group chat manager.
-            agent_states = await self._runtime.save_state()
-            return TeamState(agent_states=agent_states, team_id=self._team_id).model_dump()
+            # Store state of each agent by their name.
```

Comment:
We should consider adding to the docstring that loading state saved from a different autogen version is not supported